src/three-components/PMREMGenerator.ts

```diff
@@ -27,7 +29,9 @@ import {encodings, getDirectionChunk, getFaceChunk, texelIO} from './shader-chun
 export const generatePMREM =
     (cubeTarget: WebGLRenderTargetCube, renderer: WebGLRenderer):
         WebGLRenderTarget => {
-          const {cubeUVRenderTarget, cubeLods, meshes} = setup(cubeTarget);
+          const roughnessExtra = [0.5, 0.7, 1.0];
```

Comment:
I eventually determined that these were referring to lods, but from this line the purpose of these numbers seems kind of obscure.